.github/workflows/ci.yml

```diff
@@ -0,0 +1,92 @@
+---
+
+name: OpenWISP CI Build
```

Comment:
wouldn't be better to put the module name here? Or is it indifferent?

Reply:
This is the name which is used to refer the workflow. Also, this name is displayed on the badge on the README. I will change if you still think that will be the best thing to do.

Reply:
I think it's better to have the module name in it, so in this case `OpenWISP Controller CI Build`